app/services/katello/pulp3/repository.rb

```diff
@@ -135,7 +135,11 @@ def ssl_remote_options
       end
 
       def lookup_version(href)
-        pulp3_api.repositories_versions_read(href)
+        pulp3_api.repositories_versions_read href
+      rescue PulpcoreClient::ApiError => e
+        Rails.logger.error "Exception when calling RepositoriesApi->repositories_versions_read: #{e}"
+        nil
+      end
```

Comment:
i'm curious why this was needed?

Reply:
Probably don't need to catch exceptions here and allow method to fail in case there are errors.